pkgs/development/libraries/opencv/4.x.nix

```
@@ -472,7 +472,12 @@ effectiveStdenv.mkDerivation {
   postInstall = ''
     sed -i "s|{exec_prefix}/$out|{exec_prefix}|;s|{prefix}/$out|{prefix}|" \
       "$out/lib/pkgconfig/opencv4.pc"
-    mkdir $cxxdev
+    mkdir "$cxxdev"
+  ''
+  # Temporary fix for https://github.com/NixOS/nixpkgs/issues/276691
```

```suggestion
  # fix deps not progagating from opencv4.cxxdev if cuda is disabled
  # see https://github.com/NixOS/nixpkgs/issues/276691
```
it is always nice to know what the issue is about without needing to visit it